shell/utils/nuxt.js

```diff
@@ -331,8 +331,6 @@ export function addLifecycleHook(vm, hook, fn) {
   }
 }
 
-export const urlJoin = joinURL;
```

Comment:
No longer used after the change in `client.js`. Why even make this alias anyway?